Tests/Approval.Tests/ApprovedApi/FluentAssertions/net47.approved.txt

```diff
@@ -804,11 +800,11 @@ namespace FluentAssertions.Equivalency
         public TSelf RespectingRuntimeTypes() { }
         public TSelf ThrowingOnMissingMembers() { }
         public override string ToString() { }
-        public TSelf Using(FluentAssertions.Equivalency.IAssertionRule assertionRule) { }
         public TSelf Using(FluentAssertions.Equivalency.IEquivalencyStep equivalencyStep) { }
         public TSelf Using(FluentAssertions.Equivalency.IMemberMatchingRule matchingRule) { }
         public TSelf Using(FluentAssertions.Equivalency.IMemberSelectionRule selectionRule) { }
         public FluentAssertions.Equivalency.SelfReferenceEquivalencyAssertionOptions<TSelf>.Restriction<TProperty> Using<TProperty>(System.Action<FluentAssertions.Equivalency.IAssertionContext<TProperty>> action) { }
+        public FluentAssertions.Equivalency.SelfReferenceEquivalencyAssertionOptions<TSelf>.Restriction<TProperty> Using<TProperty>(System.Action<TProperty, TProperty> action) { }
```

Comment:
This new `Using` is not part of the generated dump.
Did that overload only exist during development of this PR?

Reply:
Yes, I need to remove it.